src/web/mjs/connectors/MangaDex.mjs

```diff
@@ -231,4 +224,26 @@ export default class MangaDex extends Connector {
         });
         return range.join('-');
     }
+
+    fetchJSON( request, retries ) {
+        retries = retries || 0;
+        if( typeof request === 'string' ) {
+            request = new Request( request, this.requestOptions );
+        }
+        // TODO: check if this will affect (replace) the input parameter?
+        if( request instanceof URL ) {
+            request = new Request( request.href, this.requestOptions );
+        }
+        return fetch( request )
+            .then( response => {
+                if( (response.status >= 500 || response.status == 404) && retries > 0 ) {
```

Comment:
From a technical point of view retrying on `404 - not found` makes no sense (the page does not exist, so why retry), maybe check something like `/captcha.js$/.test(response.url)` instead?